src/content/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/introduction-alerts.mdx

```diff
@@ -95,13 +96,13 @@ We give you control over every part of creating a robust alerting solution for y
       </td>
 
       <td>
-        We offer customizable notification channels via many common services, including email, mobile push notifications, OpsGenie, Slack, and more. To see supported services, see [Notification channels](/docs/alerts/new-relic-alerts/managing-notification-channels/notification-channels-control-where-send-alerts).
+        We offer customizable notification channels via many common services, including email, mobile push notifications, PagerDuty, Slack, and more. To get started, see [Introduction to notifications](/docs/alerts-applied-intelligence/notifications/intro-notifications).
```

Comment:
I'm pretty sure this link will work, but I haven't tested it since I wasn't completely sure how.

Reply:
Yes @dneth it will work! Thanks.